view/theme/frio/templates/admin/users.tpl

```diff
@@ -0,0 +1,266 @@
+				{{/foreach}}
+					</tbody>
+				</table>
+				<div class="panel-footer">
+					<div class="row">
+						<div class="col-md-6">
+							<div class="btn-group" role="group">
+								<button type="button" class="btn btn-default selectall" data-select-all="pending_ckbx"><i class="fa fa-check-square-o" aria-hidden="true"></i></button>
+								<button type="button" class="btn btn-default selectnone" data-select-none="pending_ckbx"><i class="fa fa-square-o" aria-hidden="true"></i></button>
+							</div>
+						</div>
+						<div class="col-md-6">
+							<button type="submit" name="page_users_deny" class="btn btn-primary"><i class="fa fa-thumbs-down" aria-hidden="true"></i> {{$deny}}</button>
+							<button type="submit" name="page_users_approve" class="btn btn-warinig"><i class="fa fa-thumbs-up" aria-hidden="true"></i> {{$approve}}</button>
+						</div>
+					</div>
+				</div>
+			{{else}}
+				<div class="panel-body text-center text-muted">{{$no_pending}}</div>
+			{{/if}}
+		</div>
+
+<!--
+	**
+	*
+	*		USERS Table
+	*
+	**
+-->
+	<div class="panel panel-default">
+		<div class="panel-heading"><h3 class="panel-title">{{$h_users}}</h3></div>
+		{{if $users}}
+
+			<table id="users" class="table table-hover">
+				<thead>
+				<tr>
+					<th></th>
+					<th></th>
+					{{foreach $th_users as $k=>$th}}
+					{{if $k < 2 || $order_users == $th.1 || ($k==5 && !in_array($order_users,[$th_users.2.1, $th_users.3.1, $th_users.4.1])) }}
+					<th>
+						<a href="{{$baseurl}}/admin/users/?o={{if $order_direction_users == "+"}}-{{/if}}{{$th.1}}">
+							{{if $order_users == $th.1}}
+								{{if $order_direction_users == "+"}}
+								&#8595;
+								{{else}}
+								&#8593;
+								{{/if}}
+							{{else}}
+								&#8597;
+							{{/if}}
+						{{$th.0}}</a>
+					</th>
+					{{/if}}
+					{{/foreach}}
+					<th></th>
+				</tr>
+				</thead>
+				<tbody>
+				{{foreach $users as $u}}
+					<tr id="user-{{$u.uid}}">
+						<td>
+						{{if $u.is_deletable}}
+							<input type="checkbox" class="users_ckbx" id="id_user_{{$u.uid}}" name="user[]" value="{{$u.uid}}"/>
+						{{else}}
+							&nbsp;
+						{{/if}}
+						</td>
+						<td><img class="icon" src="{{$u.micro}}" title="{{$u.nickname}}"></td>
+						<td><a href="{{$u.url}}" title="{{$u.nickname}}"> {{$u.name}}</a></td>
+						<td>{{$u.email}}</td>
+						{{if $order_users == $th_users.2.1}}
+						<td>{{$u.register_date}}</td>
+						{{/if}}
+
+						{{if $order_users == $th_users.3.1}}
+						<td>{{$u.login_date}}</td>
+						{{/if}}
+
+						{{if $order_users == $th_users.4.1}}
+						<td>{{$u.lastitem_date}}</td>
+						{{/if}}
+
+						{{if !in_array($order_users,[$th_users.2.1, $th_users.3.1, $th_users.4.1]) }}
+						<td>{{$u.page_flags}} {{if $u.is_admin}}({{$siteadmin}}){{/if}} {{if $u.account_expired}}({{$accountexpired}}){{/if}}</td>
+						{{/if}}
+						<td class="text-right">
+							<a href="#" onclick="return details({{$u.uid}})"><span class="caret"></span></i>
```

Comment:
Can you please replace the only remaining `<a href="#">` by a `<button type="button">` ?